UITests/Helper/Transactions/TransactionsList.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+
+class TransactionsList {
```

Comment:
We are following the naming convention of Receipt instead of Transaction even though on UI we display Transactions. So use Receipt everywhere instead of Transaction.

Reply:
fixed